### DISCUSSION TITLE
事件处理流程的有效期是多久？

### DISCUSSION THREAD

**Discussion by YangRucheng** [nonebot/discussions#335] [问答] [answered]
例如，用got装饰器接受一个参数，超过有效期就关闭整个流程，不再等待参数。

如果可以的话，我也想在过期后执行一些操作，比如发送“已超时结束对话”，或者一些异步的结束代码 await close()。

谢谢！

**Comment by yanyongyu** [comment, marked as answer]
配置里有`session_expire_time`默认两分钟，目前没法做到超时时执行hook，因为超时判断是lazy的，并不会在超时的精确时间点移除
